SWLOR.Game.Server/Feature/AbilityDefinition/Force/DisturbanceAbilityDefinition.cs

```diff
@@ -67,16 +68,18 @@ private void Disturbance1()
             _builder.Create(FeatType.Disturbance1, PerkType.Disturbance)
                 .Name("Disturbance I")
                 .Level(1)
-                .HasRecastDelay(RecastGroup.Disturbance, 20f)
-                .RequirementFP(1)
+                .HasRecastDelay(RecastGroup.Disturbance, 6f)
+                .HasActivationDelay(2f)
```

Comment:
There is no FP cost on this. Intentional?

Reply:
Nope. Gotta fix.